### PR TITLE
Improve Changelog generation performance

### PR DESCRIPTION
**Pull request by goncalvesnelson**
#### Description
This PR replaces the REST API usage with GraphQL. This would improve the speed of the CHANGELOG generation by
allowing us to fetch just the last Release and then fetching the latest PRs until the mergedAt timestamp is equal or below the createdAt timestamp of the last Release. There is however an issue, the GraphQL API currently doesn't allow us to sort the list of PRs by the mergedAt value, only by CREATED_AT or UPDATED_AT, which means, there is no reliable way to fetch the just list of PRs we need.

#### Related issues
https://github.com/octokit/graphql.js/issues/191

After the discussion in the issue above, I opened a support ticket with Github with the following text:

> Would it be possible to extend the graphQL API to allow sorting the list of pull requests by mergedAt?
> This would be very helpful in automating the creation of CHANGELOG files since it would allow us to:
> Use the API to fetch the createdAt timestamp of the last release
> Get the list of PRs sorted by mergedAt and iterate through that list, saving each PR until we find the first PR with mergedAt timestamp before the timestamp of the last release.
> This would save us from having to fetch all the PRs which can be slow on bigger repos.

To which  I got the following reply:

> Hi Nelson,
> Thank you for contacting GitHub Support.
> Thanks for your feedback! We're always working to improve GitHub and we consider every suggestion we receive.
> I have passed this on to the relevant team, although I can not promise an E.T.A, your feedback has definitely been recorded!
> Please let us know if you need any further help.

TLDR: They know we want the MERGED_AT sort but they can't promise any ETA on it.

### CODE REVIEW

**Review by Americas — APPROVED** [review]
LGTM and @goncalvesnelson
